Command/RunCommand.php

```diff
@@ -428,9 +429,25 @@ private function getCommandProcessBuilder()
             $pb->add('exec');
         }
 
+        // Localize the `console` command
```

Comment:
This doesn't look safe if the code is executed in a test case or from a web app.